tests/unit_tests/tensor-debug/meson.build

```diff
@@ -1,6 +1,8 @@
 _tests = ['tensor-debug_01.cc']
 
-_src = ['../../../src/geotop/tensor.h']
+_src = ['../../../src/geotop/tensor.h',
+        '../../../src/libraries/fluidturtle/turtle.h']
```

Comment:
you should also add the `.cc` file where the function `new_double_tensor` is defined in this `_src` variable because the **linker** complains that cannot find the symbol

if the function is just defined in the header, then you should 
```
#include <turtle.h>
```
in your test